mllib/src/test/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluatorSuite.scala

```diff
@@ -40,4 +41,51 @@ class MulticlassClassificationEvaluatorSuite
   test("should support all NumericType labels and not support other types") {
     MLTestingUtils.checkNumericTypes(new MulticlassClassificationEvaluator, spark)
   }
+
+  test("Multiclass Classification Evaluator") {
+    val labelAndScores = spark.createDataFrame(Seq(
```

Comment:
Agreed. I will add test for BinaryClassification. 
Should I close this PR right now?